docs/import/manage_disaster.js

```diff
@@ -61,48 +61,90 @@ let scoreBoundsMap;
 
 const scoreCoordinatesPath = Object.freeze([scoreCoordinatesAttribute]);
 
+const kickOffText = 'Kick off Data Processing (will take a while!)';
+const optionalWarningPrefix = '; warning: created asset will be missing ';
+
 /**
- * Checks that all fields that can be entered by the user have a non-empty
- * value. Does not check that assets actually exist, are of valid type, etc. If
- * all validation succeeds, enables kick-off button, otherwise disables and
- * changes button text to say what is missing.
+ * Checks that all mandatory fields that can be entered by the user have
+ * non-empty values. Does not check that assets actually exist, are of valid
+ * type, etc. If all validation succeeds, enables kick-off button, otherwise
+ * disables and changes button text to say what is missing.
+ *
+ * Some fields (Income, SVI) are optional. If they are missing, a separate
+ * message is displayed on the button, but it can still be enabled. If it is
+ * enabled, it is yellowed a bit to indicate the missing optional assets.
+ * The buildings asset is optional if the damage asset is not present, but is
+ * required if the damage asset is present.
  */
 function validateUserFields() {
   const states = disasterData.get(getDisaster()).states;
   const missingItems = [];
-  for (const scoreAssetType of scoreAssetTypes) {
+  const damageAssetPresent = !!$('#damage-asset-select').val();
```

Comment:
can you document what the missingItems array ends up holding before we start processing it?

Reply:
Added a comment at the top.